querybook/server/lib/ai_assistant/base_ai_assistant.py

```diff
@@ -93,6 +94,10 @@ def _get_error_msg(self, error) -> str:
 
         return str(error.args[0])
 
+    def _filter_column(self, column: DataTableColumn) -> bool:
```

Comment:
i think in filter(...), returning True means don't remove. Maybe just for clarity we can rename this to `should_skip_column`